### DISCUSSION TITLE
Ignoring a child directory inside project

### DISCUSSION THREAD

**Discussion by jcasale** [texstudio-org/texstudio#3780] [Q&A]
I have a large Latex document that has several child directories used to auto-generate some content (e.g., Excel and CSV documents, Python scripts, etc.) and this seems to slow down texstudio a lot.

I looked for a mention of ignoring directories but didn't see anything. Does anyone know how I can fix this?

**Comment by sunderme** [comment]
Txs does not scan arbitrary sub folders.
Anyway, what exactly means "slows down txs a lot" ? in what context ? What activity specifically ?

**Comment by jcasale** [comment]
When I start txs, it hangs for a long duration, and when I reload files (to revert for example), there is a long delay.

If I remove the directory, neither of those two behaviors manifest.

**Comment by sunderme** [comment]
then you need to provide an example which shows the behavior (all files).
